app/javascript/crayons/Button/__stories__/button.html.stories.jsx

```diff
@@ -46,16 +46,6 @@ Outlined.story = {
   name: 'outlined',
 };
 
-export const Text = () => (
```

Comment:
This story for the HTML components really isn't necessary. Most buttons just have text in them.